packages/components/src/dropdown-menu/stories/index.tsx

```diff
@@ -1,52 +1,29 @@
+/**
+ * External dependencies
+ */
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
 /**
  * Internal dependencies
  */
-import DropdownMenu from '../';
-import { MenuGroup, MenuItem } from '../../';
+import DropdownMenu from '..';
+import { MenuGroup, MenuItem } from '../..';
 
 /**
  * WordPress dependencies
  */
-import {
-	menu,
-	arrowUp,
-	arrowDown,
-	chevronDown,
-	more,
-	trash,
-} from '@wordpress/icons';
+import { menu, arrowUp, arrowDown, more, trash } from '@wordpress/icons';
 
-export default {
+const meta: ComponentMeta< typeof DropdownMenu > = {
 	title: 'Components/DropdownMenu',
 	component: DropdownMenu,
-	argTypes: {
-		className: { control: { type: 'text' } },
-		children: { control: { type: null } },
-		disableOpenOnArrowDown: { control: { type: 'boolean' } },
-		icon: {
```

Comment:
We may want to keep the custom `icon` controls settings, so that we display a nice dropdown instead of the raw react element object

You should be able to take inspiration from other components' storybook examples, like [Button](https://github.com/WordPress/gutenberg/blob/c160a7f2602b080b3cd55ee3e72e1825f23c4b42/packages/components/src/button/stories/index.tsx)